packages/sync/src/useSync.ts

```diff
@@ -83,6 +87,7 @@ export function useSync(opts: UseSyncOptions & TLStoreSchemaOptions): RemoteTLSt
 	const schema = useTLSchemaFromUtils(schemaOpts)
 
 	const prefs = useShallowObjectIdentity(userInfo)
+	const getUserPresence = useEvent(_getUserPresence ?? getDefaultUserPresence)
```

Comment:
not sure it's appropriate to use `useEvent` here because if the fn impl changes it won't cause the presence state signal to update reactively.

should probably do something like

```ts
const gup = _getUserPresence ?? getDefaultUserPresence
const getUserPresence$ = atom('getUserPresence', gup)
useLayoutEffect(() => getUserPresence$.set(gup), [gup])
```

Reply:
good shout!